composer.json

```diff
@@ -12,7 +12,8 @@
     "require": {
 
         "smarty/smarty": "2.6.29",
-        "adodb/adodb-php": "5.20.2"
+        "adodb/adodb-php": "5.20.2",
+        "rospdf/pdf-php": "^0.12.22"
```

Comment:
Doesn't that open the door though for developers (and production instances) to be using different versions (in the future when these are imported by developers)? Or am I being old too old fashioned :)